src/utils.js

```diff
@@ -866,6 +866,16 @@ export function matchDomain(pattern : DomainMatcher, origin : DomainMatcher) : b
     return false;
 }
 
+export function childFrameOfParentWindow(parentWindow : CrossDomainWindowType) : boolean {
```

Comment:
Check out a reproduction of the error [here](https://iframe-error.khoa.app/). Click on the `startZoidDemo` button and the error message will appear in the console.
A `childFrameOfParentWindow` function needs to be created to use in the `post-robot` repo.
This function will be used to fix the following issue: https://github.com/krakenjs/zoid/issues/315

Currently, post-robot will perform a [matchDomain](https://github.com/krakenjs/post-robot/blob/547b143553604aec3394c5a917fc64b7f46c1858/src/serialize/function.js#L83) check when attempting communication between a parent window with a `sandboxed` child window. Because the child window is sandboxed, the origin of the child window gets automatically gets set to null, which `post-robot` then [sets to file://](https://github.com/krakenjs/post-robot/blob/master/src/drivers/receive/index.js#L125) and will throw this error message template:

<img width="877" alt="Screen Shot 2020-10-13 at 5 41 44 PM" src="https://user-images.githubusercontent.com/45890848/95930474-64e80d80-0d7b-11eb-8751-d1ca4b14fa4f.png">